front/app/containers/Admin/dashboard/overview/charts/EventsCard/config.test.ts

```diff
@@ -2,7 +2,7 @@ import moment from 'moment';
 
 import { Query } from 'api/analytics/types';
 
-import { StatCardData } from '../StatCard/useStatCard/typings';
+import { StatCardData } from '../../../../../../components/admin/GraphCards/StatCard/useStatCard/typings';
```

Comment:
Minor and probably preference but for these longer links we could use absolute imports. Same for the other ones

```suggestion
import { StatCardData } from 'components/admin/GraphCards/StatCard/useStatCard/typings';
```